app/ui/browser2/views/browser-content.jsx

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
This looks like it should be shared? I'd expect the content area to be a set of Pages, and that is the same in every frontend I can imagine.